Test/git-issues/git-issue-3804b.dfy.expect

```diff
@@ -0,0 +1,10 @@
+git-issue-3804b.dfy(15,9): Error: a postcondition could not be proved on this return path
+git-issue-3804b.dfy(17,12): Related location: this is the postcondition that could not be proved
+git-issue-3804b.dfy(31,12): Error: a postcondition could not be proved on this return path
+git-issue-3804b.dfy(31,5): Related location: this is the postcondition that could not be proved
```

Comment:
I expect that this is _not_ an error. The reason is that the `by method` part should only need to establish that it returns the same value as the function body. It is the function body's responsibility to ensure that the function's `ensures` clauses follow from the function body.

Reply:
> I expect that this is not an error. The reason is that the by method part should only need to establish that it returns the same value as the function body. It is the function body's responsibility to ensure that the function's ensures clauses follow from the function body.

As explained above, it's an error because we cannot prove here the precondition of calling the function since the precondition is opaque.